addons/sale_stock/models/account_move.py

```diff
@@ -29,7 +29,7 @@ def _get_invoiced_lot_values(self):
         if self.state == 'draft' or not self.invoice_date or self.type not in ('out_invoice', 'out_refund'):
             return []
 
-        current_invoice_amls = self.invoice_line_ids.filtered(lambda aml: not aml.display_type and aml.product_id and aml.quantity)
+        current_invoice_amls = self.invoice_line_ids.filtered(lambda aml: not aml.display_type and aml.product_id and aml.product_id.type == 'product' and aml.quantity)
```

Comment:
```suggestion
        current_invoice_amls = self.invoice_line_ids.filtered(lambda aml: not aml.display_type and aml.product_id and aml.product_id.type in ('consu', 'product') and aml.quantity)
```

A kit could be a consumable product and represent several storable components

Reply:
(And therefore you will have to update the commit message)